.sage/main.go

```diff
@@ -73,7 +72,10 @@ func GoLintFix(ctx context.Context) error {
 
 func GoFormat(ctx context.Context) error {
 	sg.Logger(ctx).Println("formatting Go files...")
-	return sggolines.Run(ctx)
+	return sggolangcilintv2.Fmt(
```

Comment:
Corresponding function should be `FixFix` as `sggolines.Run` wrote changes.
```suggestion
	return sggolangcilintv2.FmtFix(
```

Reply:
Done!